### PR TITLE
fix: Cleanup path resolves

### PR DESCRIPTION
**Pull request by salisdemos**
This path to the validator pointed to the root instead of the propper validation folder

### CODE REVIEW

**Review by muellerdemos — APPROVED** [review]
lgtm, thanks

File: lib/validation/dpValidateMultiselectDirective.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { validateMultiselect } from '../../index'
+import validateMultiselect from '../../lib/validation/utils/validateMultiselect'
```
Comment (hwiem) [inline lib/validation/dpValidateMultiselectDirective.js]: This looks unnecessarily complicated. Can't we do
`import { validateMultiselect } from './index.js'`
or
`import validateMultiselect from './utils/validateMultiselect'`?

**Review by hwiem — DISMISSED** [review]

File: lib/validation/dpValidateMultiselectDirective.js (hunk 1 above)
Comment (salisdemos) [reply lib/validation/dpValidateMultiselectDirective.js]: :see_no_evil:  [1586976](https://github.com/demos-europe/demosplan-js-utils/pull/7/commits/1586976261d8930eed48f3d89ff6050068fb10e5)

Comment (muellerdemos) [reply lib/validation/dpValidateMultiselectDirective.js]: I admit I had to giggle a bit ;)

**Review by muellerdemos — APPROVED** [review]
@hwiem concerns were addressed and fixed 
approviig since @hwiem is not in the office today
- lgtm

**Pull request merged**